bugzilla.py

```diff
@@ -0,0 +1,12 @@
+import utils
+
+
+def get_signatures_from_bug(bug_id):
+    url = 'https://bugzilla.mozilla.org/rest/bug'
+    response = utils.get_with_retries(url, params={'id': bug_id})
+    response.raise_for_status()
+    return [str(sig)[1: len(sig) - 1] for sig in response.json()['bugs'][0]['cf_crash_signature'].split('\r\n')]
```

Comment:
We should remove the `[@ ` and `]` characters from the signature. There are several options, as the format is not always the same:
1. [@ SIGNATURE ]
2. [@SIGNATURE]
3. [@ SIGNATURE]
4. [@SIGNATURE ]

So, for each string, we should remove `[@` and `]` from the beginning and end of the string and then trim the result.